rules/scalafmt/scalafmt/ScalafmtRunner.scala

```diff
@@ -35,12 +36,14 @@ object ScalafmtRunner extends WorkerMain[Unit] {
       if (code == formatted) code else format(formatted)
     }
 
+    val colorMessage = new ColorMessage()
```

Comment:
This isn't needed if we make `ColorMessage` a singleton.

Reply:
Good idea, fixed.